test/plugins/poster_spec.js

```diff
@@ -95,4 +95,13 @@ describe('Poster', function() {
     expect(this.poster.shouldHideOnPlay()).to.equal(false)
   })
 
+  it('renders custom background', function() {
```

Comment:
Cool! Now, we (clappr) also need to add a test case for `background-image` 👍 

Nice work @NicholasAsimov